react-native/react/actions/tracker.js

```diff
@@ -9,16 +9,14 @@ import {flattenCallMap, promisifyResponses} from '../engine/call-map-middleware'
 import type {State as RootTrackerState} from '../reducers/tracker'
 
 import type {CallMap} from '../engine/call-map-middleware'
-import type {UserSummary} from '../constants/types/flow-types'
 import type {Action, Dispatch} from '../constants/types/flux'
-import type {UserInfo} from '../tracker/bio.render.types'
 
-import type {Identity, IdentifyKey, TrackSummary, User, Cryptocurrency, IdentifyOutcome, RemoteProof, LinkCheckResult, TrackOptions} from '../constants/types/flow-types'
+import type {Identity, IdentifyKey, TrackSummary, User, Cryptocurrency, IdentifyOutcome, RemoteProof, LinkCheckResult, TrackOptions, UserCard} from '../constants/types/flow-types'
 
 // TODO make actions for all the call back stuff.
 
 export function registerIdentifyUi (): (dispatch: Dispatch) => void {
```

Comment:
this type isn't correct, but you can use the newly merged TrackerActionCreator: https://github.com/keybase/client/pull/1479/files#diff-49efa58b5e3f2f865d5c952996d692b3R18